packages/base/src/util/TabbableElements.js

```diff
@@ -27,6 +27,10 @@ const getTabbables = (nodes, tabbables) => {
 			currentNode = Array.from(children).find(node => node.tagName !== "STYLE");
 		}
 
+		if (!currentNode) {
+			return;
```

Comment:
It might be more safe to return empty array - `return []`,
as the current code seems to expect an array:


```js
// List.js
const aContent = getTabbableElements(this.getDomRef());
!aContent.length

// Table.js
row._tabbableElements = getTabbableElements(row);
row._tabbableElements.forEach
```